include/crm/stonith-ng.h

```diff
@@ -692,6 +692,18 @@ const char *stonith_action_str(const char *action);
  * we have to duplicate these declarations where they're implemented.
  */
 
+//! \deprecated Do not use
+#define T_STONITH_NOTIFY_DISCONNECT "st_notify_disconnect"
```

Comment:
I remember seeing the note at the top around the time I added these deprecations... I don't remember if I
* forgot before actually doing these; or
* justified it as "`stonith-ng.h` doesn't actually include it, so it's fine."

Seems more likely I forgot, because the compat header isn't included anywhere else either. External code would have to include it directly.

Darn.